libflux/include/influxdata/flux.h

```diff
@@ -27,6 +27,9 @@ void flux_free_error(struct flux_error_t *);
 // becomes invalid when the error is freed.
 const char *flux_error_str(struct flux_error_t *);
 
+// Prints the flux error to stdout
+const char *flux_error_print(struct flux_error_t *);
```

Comment:
This returns a char* while the Rust implementation doesn't return anything and the Go code doesn't expect a return. Can you update this signature to match?

Do we want to print to stdout from Rust? Or should we make this method a `flux_error_format` instead that builds a string of the contents and let the caller decide to print to stdout. This would make UI errors more possible for example.

Reply:
> Do we want to print to stdout from Rust? Or should we make this method a flux_error_format instead that builds a string of the contents and let the caller decide to print to stdout. This would make UI errors more possible for example.

Building a string and returning it only works for for ANSI compatible terminals so this seemed more general to me. It also auto detects if coloring is supported.